src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -395,6 +406,9 @@ type VehicleParking implements Node & PlaceInterface {
     The currently available spaces at this vehicle parking.
     """
     availability: VehicleParkingSpaces
+
+    """Opening hours of the parking facility"""
+    openingHours: OpeningHours
```

Comment:
I was a bit hesitant to add the opening hours schema I quickly came up with for otp1 to this new vehicle parking type because I wasn't sure if the structure made any sense. This now kinda adds it here but I don't think it's actually a problem because we can make breaking changes again when we build the new API.

Reply:
Would you prefer to remove to remove the OTP1 opening hours from the entire schema?

Reply:
That way you can define something when you're ready.

Reply:
Nah I think you can leave it there for now. I'm not sure yet what we will do about those.